phantom-dsl/src/main/scala/com/websudos/phantom/util/ByteIterator.scala

```diff
@@ -184,8 +188,8 @@ object ByteIterator {
     }
     normalize()
 
-    @inline private def current: ByteArrayIterator = iterators.head
-    @inline private def dropCurrent(): Unit = { iterators = iterators.tail }
+    @inline private[this] def current: ByteArrayIterator = iterators.head
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Usage of head on collections.](https://www.codacy.com/app/flavian/phantom_2/file/2179334561/issues/source?bid=2847375&fileBranchId=3164226#l191)